lib/active_utils/network_connection_retries.rb

```diff
@@ -10,18 +10,23 @@ module NetworkConnectionRetries
       OpenSSL::SSL::SSLError => "The SSL connection to the remote server could not be established"
     }
 
+    DEFAUT_RETRY_ERRORS = {
+      Errno::ECONNREFUSED => "The remote server refused the connection"
+    }
+
     def self.included(base)
       base.send(:attr_accessor, :retry_safe)
     end
 
     def retry_exceptions(options={})
       connection_errors = DEFAULT_CONNECTION_ERRORS.merge(options[:connection_exceptions] || {})
+      retry_errors = DEFAUT_RETRY_ERRORS.merge(options[:retryable_exceptions] || {})
```

Comment:
You made me look into this, because my intuition would've been to drop the `y`:
	https://english.stackexchange.com/questions/305273/retriable-or-retryable

But looks like both work.

Reply:
`Retriable` is wrong. It is a false syllogism stemming from the prevalence of English nouns / verbs that end in `y`.

```
envy -> enviable
vary -> variable
levy -> leviable
pity -> pitiable
```

But note the difference in pronunciation. Each of the above cases where `y` -> `i` substitution is performed legitimately feature a hard `E` sound on the trailing syllable. This is true of most two-syllable English words that end with `Y`. However, `try` has a hard `I` sound, as is true of most one-syllable English words ending with `Y`. In order to denote the preservation of that hard `I` sound in their pronunciation, single-syllable verbs retain the `Y` when presented in the instrumental case.